### ISSUE TITLE
[SCREENREADER]: GIBCT EYB - CONSIDER updating the behavior for announcing changes to user benefits

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#10199]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

After reviewing the existing flow that sets focus on the Estimate Your Benefits heading after users press `Update your benefits` with the design team, we agreed on a new approach. This new approach would use an aria `live-region` and a focus-only link to allow users to skip to the benefits panel OR keep updating filters. 

I mocked up the proposed user flow in a Codepen here: https://codepen.io/tpierce_402/pen/BajKREW

This is not meant to be prescriptive, but just an example of how we could achieve the goals identified in our meeting on 6/11.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] User flow meets the BAH design team's stated goals
- [ ] New code throws no axe errors
- [ ] Skip link only appears on `:focus`

## WCAG or Vendor Guidance (optional)

* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

**Comment by hughes-dustin** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/13204
Branch: bah-10199

**Comment by hughes-dustin** [comment]
@1Copenut Where should the focus go after we click on the  "Update Benefits" button? We can't focus back to the button because it becomes disabled after clicking on it.

**Comment by 1Copenut** [comment]
@hughes-dustin My hunch is the focus should go to the open sub-accordion button in this case. That should be tested as much as possible with users to confirm it's an okay expectation.

**Comment by delli-gatti-michael** [comment]
local testing complete

**Comment by delli-gatti-michael** [comment]
This is now in Staging

**Comment by desireeturner** [comment]
@1Copenut this is ready for review.

**Comment by 1Copenut** [comment]
@desireeturner I took a look at this briefly in staging. I'd like to take another look, and focus on two improvements:

1. I'd like to set focus on the expand/collapse button after users press `Update benefits`. Right now we are setting focus on the container, and that's not providing a lot of context.
2. The gray benefits panel is not being read aloud to screen readers. I'm almost certain this can be adjusted with a small markup change.

I'm back and working normal hours, so I'll follow up with you tomorrow.

**Comment by 1Copenut** [comment]
@hughes-dustin Figured a couple of screenshots were the easiest way to communicate changes async. LMK if these are unclear and we can follow up.

---
![Screen Shot 2020-07-01 at 9.49.25 AM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/b3818f5d-b240-431d-9405-b7c99b4941f5)

---
![Screen Shot 2020-07-01 at 9.55.32 AM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/429a3b0b-d2cd-4b2a-9c85-4ac17e01b760)

**Comment by hughes-dustin** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/13349
Branch: bah-10199-b

**Comment by hughes-dustin** [comment]
Aria-live polite will wait for the current thing to stop (in this case the focus to the sub-accordion button), then read the live region (the Estimate Benefits fields being updated).

**Comment by sonntag-adam** [comment]
@1Copenut in staging.

**Comment by 1Copenut** [comment]
@desireeturner && @sonntag-adam && @hughes-dustin 
I reviewed this one in staging using JAWS, NVDA, and VoiceOver with their respective browser pairings. Things appear to be working correctly, except in JAWS + IE11. Focus is being moved to the correct place; the live region is not announcing the changed data values.

I'm going to pass this issue and open two new ones, one for the IE11 live region not working as expected, and one to change the skip to your estimated benefits to a link. IE11 doesn't like that being a button, and it doesn't move the user to the H3 as it should.
